buildSrc/src/main/kotlin/io/spine/internal/dependency/ProtoData.kt

```diff
@@ -65,7 +65,7 @@ object ProtoData {
      * The version of ProtoData dependencies.
      */
     val version: String
-    private const val fallbackVersion = "0.21.3"
+    private const val fallbackVersion = "0.30.0"
```

Comment:
I think we have a much newer version.

Also, I am not sure why we update the dogfooding version to some different value, than this.

Please review.

Reply:
It came from `master` in `config`. I've updated it to latest ProtoData. It's not used by Base, but let's bump it anyway.